website/www/site/content/en/get-started/quickstart/java.md

```diff
@@ -0,0 +1,195 @@
+---
+title: "Beam Quickstart for Java"
+aliases:
+  - /get-started/quickstart/
+  - /use/quickstart/
+  - /getting-started/
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Apache Beam Java SDK quickstart
+
+This quickstart shows you how to run an
+[example pipeline](https://github.com/apache/beam-starter-java) written with
+the [Apache Beam Java SDK](/documentation/sdks/java), using the
+[Direct Runner](/documentation/runners/direct/). The Direct Runner executes
+pipelines locally on your machine.
+
+If you're interested in contributing to the Apache Beam Java codebase, see the
+[Contribution Guide](/contribute).
+
+On this page:
+
+{{< toc >}}
+
+## Set up your development environment
+
+Use [`sdkman`](https://sdkman.io/) to install the Java Development Kit (JDK).
+
+{{< highlight >}}
```

Comment:
These highlight blocks have newlines before and after the listing making it look funny. Can they be removed?

Reply:
Maybe changing them to backtick blocks ` ``` ` would help? Maybe the `{{< highlight >}}` tag includes the newlines kind of like how `<code>` tags in HTML work?

Reply:
If you mean in the rendered page, that seems to be the built-in styling (e.g. compare https://beam.apache.org/get-started/quickstart-java/#get-the-example-code)

Reply:
Looking at the [staged page](http://apache-beam-website-pull-requests.storage.googleapis.com/22747/get-started/quickstart/java/index.html)
it looks like the backtick blocks ` ``` ` do remove the extra spaces, but the "copy contents" button at the top right disappear.

I don't think it's a big deal having the extra spaces. One thing to try could be having everything in the same line, but that would make the markdown a lot uglier.

```
-- Instead of
{{< highlight >}}
code here
{{< highlight >}}

-- Would be something like this
{{< highlight >}}code here{{< highlight >}}
```

I suspect it might be parsing the tags like HTML tags. That could be a workaround but it could be pretty ugly. I'm okay with having the extra spaces and having cleaner code. What are your thoughts on it?

Reply:
my $0.02: The existing docs also render like this, and use the same markup, so I'd vote to keep it this way for consistency. (Seems like an issue with the highlight.html shortcode rather than the markup in the doc.)

Reply:
Yeah, no problem. Sounds good to me.

Reply:
@davidcavazos  Is there anything else needed on this PR or is it OK to merge?

Reply:
I think it's good to merge. This is great, thank you!